spec/services/catalog/order_state_transition_spec.rb

```diff
@@ -0,0 +1,81 @@
+describe Catalog::OrderStateTransition do
+  let(:tenant) { create(:tenant) }
+  let!(:order) { create(:order, :tenant_id => tenant.id) }
+
+  let!(:order_item) { create(:order_item, :order_id => order.id, :portfolio_item_id => "1", :tenant_id => tenant.id) }
+  let!(:order_item2) { create(:order_item, :order_id => order.id, :portfolio_item_id => "1", :tenant_id => tenant.id) }
+
+  describe "#transition_state" do
```

Comment:
This should probably be `describe "#process" do` since `#transition_state` doesn't exist.

Reply:
Whoops, copy/paste error. Fixed!